facebook_connector.py

```diff
@@ -0,0 +1,41 @@
+import os
+import fnmatch
+import pandas as pd
+from fbchat_archive_parser.parser import MessageHtmlParser
+
+def initialize(location = '.'):
+    global _message_file
+    fb_message_filename = 'messages.htm'
+    _message_file = os.path.join(location, fb_message_filename)
+    if os.path.isfile(_message_file):
+        return
+    count = 0
+    fb_archive_path = 'html'
+    for file in os.listdir(location):
```

Comment:
Is it ever the case that the file isn't directly beneath the root?  Why would that be?  And if it is you should use os.walk() to recursively look for it instead of this approach, it'll be clearer.  Also instead of fnmatch just use regexes with the re package.